via/app.py

```diff
@@ -40,9 +41,27 @@ def create_app(_=None, **settings):
     config.include("via.views")
     config.include("h_pyramid_sentry")
 
+    # Configure Pyramid so we can generate static URLs
+    static_path = resource_filename("via", "static")
+    cache_buster = PathCacheBuster(static_path)
+    print(f"Cache buster salt: {cache_buster.salt}")
```

Comment:
Should the print statement be here?

Reply:
It's nearly impossible to test if you don't know the salt, as it's essentially a random number. You can get it by making a call, inspecting the contents of the HTML etc. but I think this is useful boot information if you are trying to debug it.

Should this be a `logging.get_logger()` log message instead?

Reply:
I tried this with logger but it doesn't show up. I think this is because the pyramid system hasn't loaded yet. It's useful stuff so I'm going to leave it for now.